src/decode.c

```diff
@@ -306,6 +306,87 @@ static inline uint16_t c_decode_cbtype_imm(const uint16_t insn)
 }
 #endif /* RV32_HAS(EXT_C) */
 
+#if RV32_HAS(EXT_V) /* RV32_HAS(EXT_V) */
```

Comment:
Replace the comment `/* RV32_HAS(EXT_V) */` with something like "Vector extension."